### DISCUSSION TITLE
Visual Theme Editor

### DISCUSSION THREAD

**Discussion by ivend** [primefaces/community#1775] [PrimeReact]
Hi
As per the published roadmap - the Theme Editor is expected to be released in Q2.
Can you please confirm by when can we likely have access to this and other Q2 deliverables - May 2024 ?
Is it possible to have access to the Beta versions?

**Comment by melloware** [comment]
As far as I know its still on the roadmap but I have not seen any updated timelines. They are very busy working on Primereact Tailwind support right now.

**Reply by ivend** [reply]
What are the release timelines for Primereact Tailwind support?

**Reply by melloware** [reply]
Not sure about the timeline other than its the next thing they are working on
